### PR TITLE
Add documents for how to download jcli from bintray

### PR DESCRIPTION
**Pull request by LinuxSuRen**
**Make sure that you've checked the boxes below before you submit PR:**

### Always

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Written well with PR title, we generate the [release notes](https://github.com/jenkins-zh/jenkins-cli/releases) base on that

### For the bug fixes or features only

- [ ] [Quality Gate Passed](https://sonarcloud.io/dashboard?id=jenkins-zh_jenkins-cli). Change this URL to your PR.
- [ ] The coverage is xxx on the new lines
- [ ] I've tested it locally

### CODE REVIEW

File: doc/download-zh.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## JFrog Bintray
+你可以从 [bintray.com/jenkins-zh](https://bintray.com/jenkins-zh/jenkins-cli/jenkins-cli) 下载 `jcli`.
+
+`curl -L "https://bintray.com/jenkins-zh/jenkins-cli/download_file?file_path=v0.0.24%2Fjcli-darwin-amd64.tar.gz"|tar xzv`
+
+## 镜像
+如果您无法从 GitHub 上下载 `jcli`，请尝试下面的方法：
```
Comment (yJunS) [inline doc/download-zh.md]: 这句话感觉跟这里的上下文有些不符，这块内容并没有提到从 GitHub 上下载，我看到这段是从 `README.md` 里复制的，是否可以将这段话进行修改或者去除？（`download.md` 内相同情况）

Comment (LinuxSuRen) [reply doc/download-zh.md]: 默认是从 GitHub 的 release 上下载的

Comment (yJunS) [reply doc/download-zh.md]: 了解。不过之前是在只有 GitHub 的 release 基础上下载的情况下，现在添加了 `bintray.com` 的下载方式，这样下来 GitHub 并不是唯一的选择了。所以感觉有些不严谨。

Comment (LinuxSuRen) [reply doc/download-zh.md]: 有道理，做了新的调整。

**Pull request merged**